### ISSUE TITLE
Non-json text outputted while using json formatter

### ISSUE THREAD

**Issue by BobReid** [microsoft/pyright#1006]
**Describe the bug**
Non-json output timing diagnostics are printed to standard out when using json formatter which make parsing difficult. 

**To Reproduce**
run ```pyright --outputjson  test.py```

test.py defined as:
```py
def foo():
    pass


foo(1)
```

The last line printing to standard out is ```Completed in 0.726sec```

**Expected behavior**
Only valid json is printing to standard out so it can be parsed by other tools. 

**VS Code extension or command-line**
Command line
Version: pyright 1.1.65

**Comment by erictraut** [comment]
Thanks for reporting the problem. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.67, which I just published.
